### PR TITLE
Redo new member flow

### PR DESCRIPTION
**Pull request by DarrylBrooks97**
1. New member accepts community guidelines (Receiving `onboarding` role)
2. Member is messaged in the `onboarding` channel with instructions on using `/intro` command
3. Member uses `/intro` command. The `onboarding` role is removed and the `member` role is added

Notes: 
- New members (no roles) only have access to community guidelines
- Members with `onboarding` role can only see the `onboarding`, and `community-guidelines` channels
- Members can see all channels except `on-boarding` channel

### CODE REVIEW

File: src/commands/intro.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import { CommandInteraction, CacheType, GuildMember } from 'discord.js';
+import { CommandInteraction, CacheType, GuildMember, TextChannel, Message } from 'discord.js';
```
Comment (Genyus) [inline src/commands/intro.ts]: What are these new imports needed for?

**Review by Genyus — CHANGES_REQUESTED** [review]
Looks good generally, just has a couple stray imports in intro.ts that I'm guessing might've been added before refactoring

File: src/commands/intro.ts (hunk 1 above)
Comment (DarrylBrooks97) [reply src/commands/intro.ts]: Didn't see this. Not needed at all removing them  now

**Review by Genyus — APPROVED** [review]
GTG

**Pull request merged**